l10n_pa_withholding/__openerp__.py

```diff
@@ -0,0 +1,24 @@
+# coding: utf-8
+
+{
+    "name": "Panama Withholding",
+    "version": "8.0.0.1.0",
+    "author": "Vauxoo",
+    "category": "Generic Modules",
+    "website": "http://www.vauxoo.com/",
+    "license": "AGPL-3",
+    "depends": [
+            "l10n_pa_reports",
```

Comment:
Why depend of reports? I was expecting this as a procedural modules only, in all case the one that should depend of this one is the l10n_pa_report